src/rollback_to_stable/rts_btree.c

```diff
@@ -750,7 +750,7 @@ __rts_btree_abort_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_ROW *rip,
                 upd->txnid = tw->start_txn;
             upd->durable_ts = tw->durable_start_ts;
             upd->start_ts = tw->start_ts;
-            F_SET(upd, WT_UPDATE_RESTORED_FROM_DS);
+            F_SET(upd, WT_UPDATE_DURABLE | WT_UPDATE_RESTORED_FROM_DS);
```

Comment:
We should also set flags in these cases though we should not hit them in disagg.